Readme.md

```diff
@@ -203,8 +203,8 @@ that you can start or bind or map or whatever you want to do. To handle errors,
 for instance, we could do the following:
 
 ```swift
-provider = ReactiveSwiftMoyaProvider<GitHub>()
-provider.request(.userProfile("ashfurrow")).start { event in
+provider = MoyaProvider<GitHub>()
```

Comment:
Wasn't sure about it. This way you can more easily switch (but why?), if you use the suffix immediately you can't forget it by mistake.